notebooks/test_wps_resolve_rules_errors.ipynb

```diff
@@ -0,0 +1,136 @@
+{
```

Comment:
<p>Could you catch the exception here and assert that we are getting it rather than leaving it as is?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pacificclimate/sandpiper/pull/55/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>